recipes/jsonnet/all/conanfile.py

```diff
@@ -38,11 +38,12 @@ def config_options(self):
     def configure(self):
         if self.options.shared:
             del self.options.fPIC
+            self.options["rapidyaml"].shared = True
```

Comment:
why?

Reply:
Sorry for late response.
I missed this PR.

I make rapidyaml shared to avoid compilation error like following.
https://c3i.jfrog.io/c3i/misc/logs/pr/9786/2-linux-gcc/jsonnet/0.18.0//9661504392a1e1eb6eddbd4543cc0efb2f0b1bad-test.txt

Please tell me if you have better way to solve this.

Reply:
It means that there is something wrong somewhere (in rapidyml maybe), so it would be better to find the root cause of this issue.

Reply:
Thank you for comment.
I try to solve this.

Reply:
@SpaceIm 
I tried several ideas, but I can't solve it.
I am going to make rapidyaml shared with some comments.